includes/degree-functions.php

```diff
@@ -470,27 +470,31 @@ function get_degree_application_deadline_modern_layout( $degree ) {
  * @return string HTML markup
  */
 function get_degree_course_overview_modern_layout( $degree ) {
+	$raw_postmeta        = get_post_meta( $degree->ID );
+	$post_meta           = format_raw_postmeta( $raw_postmeta );
+
 	$course_overview = get_field( 'course_overview', $degree );
+	$course_catalog_link_text = get_field( 'course_catalog_link_text', $degree );
 
 	if ( empty( $course_overview ) ) return '';
 
 	ob_start();
 
 	if ( have_rows( 'course_overview', $degree ) ) : ?>
-		<div class="py-4">
-			<div class="container py-lg-3">
+		<section aria-label="Course Overview">
+			<div class="container py-lg-3 my-5">
 				<div class="row my-lg-3">
 					<div class="col-12">
 						<h2 class="font-condensed text-uppercase mb-4">Course Overview</h2>
 						<div class="accordion" role="tablist" id="courses">
 							<?php while ( have_rows( 'course_overview', $degree ) ) : the_row(); ?>
 								<div class="accordion-courses mt-0 pt-0 pt-lg-3">
 									<h3 class="mb-0">
-										<a data-toggle="collapse" data-target="#course-<?php echo get_row_index(); ?>" aria-controls="course-<?php echo get_row_index(); ?>" role="button" tabindex="0" aria-expanded="true">
+										<a <?php if ( get_row_index() !== 1 ) echo 'class="collapsed"' ?> data-toggle="collapse" data-target="#course-<?php echo get_row_index(); ?>" aria-controls="course-<?php echo get_row_index(); ?>" role="button" tabindex="0" aria-expanded="true">
```

Comment:
Wrapping the entire `<h3>` in this link would make the link block-level, and make the accordion items easier to click.